### ISSUE TITLE
Add Additional Font Settings in the Global Styles

### ISSUE THREAD

**Issue by schahramnouri** [soflyy/oxygen-bugs-and-features#1861]
Hi,
Something very important is missing in the Global Styles.
1- Add Letter Spacing Setting for Global Styles/Headings
2- Add Letter Spacing Setting for Global Styles/Fonts
3- Add Mobile Settings for the Global Styles/Headings and Global Styles/Fonts
For example, the font size, font height, font weight, and letter spacing could be added to 4 screen size categories. This would allow to setup fonts for all kind of screen sizes and they could be updated globally instead of creating classes. 

Thank you so much,
Amir

**Comment by jamescrabb** [comment]
I agree there needs to be great control over global styles

**Comment by nickaharding** [comment]
I agree all of this would be nice, but particularly Global styles for different screen sizes.
